packages/ts/react-grid/test/autocrud.spec.tsx

```diff
@@ -131,6 +131,61 @@ describe('@hilla/react-grid', () => {
       expect(dialog?._getDialogText()).to.equal('Are you sure you want to delete the selected item?');
       expect('Dove').to.equal(getBodyCellContent(grid, 1, 1).innerText);
     });
+    it('can add a new item', async () => {
+      const service = personService();
+      const result = render(<ExperimentalAutoCrud service={service} model={PersonModel} />);
+      const grid: GridElement<Person> = getGrid(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      const newButton = await result.findByText('+ New');
+      newButton.click();
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      expect((await getFormField(result, 'First name')).value).to.equal('');
+      expect((await getFormField(result, 'Last name')).value).to.equal('');
+      expect((await getFormField(result, 'Some number')).value).to.equal(0);
+      await setFormField(result, 'First name', 'Jeff');
+      await setFormField(result, 'Last name', 'Lastname');
+      await setFormField(result, 'Email', 'first.last@domain.com');
+      await setFormField(result, 'Some number', '12');
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      await submit(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      expect(getBodyCellContent(grid, 1, 0).innerText).to.equal('Jeff');
+    });
+    it('can update added item', async () => {
+      const service = personService();
+      const result = render(<ExperimentalAutoCrud service={service} model={PersonModel} />);
+      const grid: GridElement<Person> = getGrid(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      const newButton = await result.findByText('+ New');
+      newButton.click();
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      await setFormField(result, 'First name', 'Jeff');
+      await setFormField(result, 'Last name', 'Lastname');
+      await setFormField(result, 'Email', 'first.last@domain.com');
+      await setFormField(result, 'Some number', '12');
+      await submit(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      await setFormField(result, 'First name', 'Jerp');
+      await submit(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      expect(getBodyCellContent(grid, 1, 0).innerText).to.equal('Jerp');
```

Comment:
Could also assert that the number of items / rows doesn't increase.

Reply:
Added